docs/Push-Server-API.md

```diff
@@ -1,4 +1,5 @@
 # Push Server API
+<!-- template api -->
```

Comment:
I think that this line has no effect.

Reply:
It does - it selects the correct page template in the documentation portal.